server/src/main/java/org/schoellerfamily/pathfinderhelper/datamodel/SpellsByLevel.java

```diff
@@ -0,0 +1,98 @@
+package org.schoellerfamily.pathfinderhelper.datamodel;
+
+import java.util.ArrayList;
+
+public class SpellsByLevel
+{
+	private ArrayList<Spell> zerothlevelspells;
+	private ArrayList<Spell> firstlevelspells;
+	private ArrayList<Spell> secondlevelspells;
+	private ArrayList<Spell> thirdlevelspells;
+	private ArrayList<Spell> fourthlevelspells;
+	private ArrayList<Spell> fifthlevelspells;
+	private ArrayList<Spell> sixthlevelspells;
+	private ArrayList<Spell> seventhlevelspells;
+	private ArrayList<Spell> eighthlevelspells;
+	private ArrayList<Spell> ninthlevelspells;
+	public ArrayList<Spell> getZerothlevelspells()
+	{
+		return zerothlevelspells;
+	}
+	public void setZerothlevelspells(ArrayList<Spell> zerothlevelspells)
```

Comment:
* Instead of setters for lists, it might be better to expose list syntax (add, addAll, remove) for each of the lists. This might not play well with the persistence framework.
* Is order important?
* Are spells allowed to be duplicated in any given list?